near-sdk-macros/src/core_impl/abi/abi_generator.rs

```diff
@@ -146,6 +154,18 @@ impl ImplItemMethodInfo {
                 }
             };
         }
+        let params = match self.attr_signature_info.input_serializer {
+            SerializerType::JSON => quote! {
+                near_sdk::__private::AbiParameters::Json {
```

Comment:
That would be nice, especially since we had to intentionally disable clippy warning about uppercase acronyms because of this. This is a part of the near-sdk-macros public API, but presumably it does not actually matter for backwards compatibility?